src/csharp/Pester/DebugConfiguration.cs

```diff
@@ -34,6 +34,7 @@ public static DebugConfiguration ShallowClone(DebugConfiguration configuration)
             WriteDebugMessages = new BoolOption("Write Debug messages to screen.", false);
             WriteDebugMessagesFrom = new StringArrayOption("Write Debug messages from a given source, WriteDebugMessages must be set to true for this to work. You can use like wildcards to get messages from multiple sources, as well as * to get everything.", new string[] { "Discovery", "Skip", "Mock", "CodeCoverage" });
             ShowNavigationMarkers = new BoolOption("Write paths after every block and test, for easy navigation in VSCode.", false);
+            ShowStartMarkers = new BoolOption("Write an indication when each test starts.", false);
```

Comment:
This should probably go into output rather than debug? wdyt @fflaten ?

Reply:
Sure, moved to Output.